code/src/components/MovieCard.css

```diff
@@ -0,0 +1,33 @@
+.card-image {
+  width: 100%;
+  height: 100%;
+}
+
+.title-date {
+  display: none;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding-left: 20px;
+  color: white;
+}
+
+.card-image:hover {
+  opacity: 0.3;
+}
```

Comment:
Here you could have instead made the hover effect on the whole link (and added a class and hover effect on that one) and then put a rgba background color with opacity in .title-date. Cause now when hovering over the links the effect disappears when you get over the text.